lib/game_of_life_web/templates/universe/show.html.leex

```diff
@@ -1,3 +1,9 @@
+<style>
+.cell.alive {
+  background: <%= render_color_rgb(@color) %>;
+}
+</style>
```

Comment:
I was initially setting the `%Color{}` struct on the `%Universe{}` itself, and I was going to use that in `render_universe` to set the color inline on each cell, but it actually felt much cleaner to just add this `<style>` tag here and let the browser style engine do more of the work.

Reply:
I really like this solution!